### DISCUSSION TITLE
wo ist vulva problem 2.bild

### DISCUSSION THREAD

**Discussion by alisia02** [Lobby-Hoes/wo-ist-vulva#19] [Ideas]
in dem 2. level von wo ist vulva, da ist die vulva hinter dem grauen bereich, wo man nicht hinklicken kann. kann man das fixen, oder ist das nicht möglich?

**Comment by eleander56-b** [comment]
Das wollte eigentlich schonmal jemand fixen... Ich weiß aber leider nicht mehr wer. Ich gucke es mir bei Gelegenheit nochmal an. Danke dir!
